beta/nncf/tensorflow/graph/converter.py

```diff
@@ -190,3 +143,143 @@ def get_graph_to_layer_var_names_map(concrete_fun):
                 nxmodel.add_edge(input_node_name, node_name)
 
     return nxmodel
+
+
+def convert_keras_model_to_nncf_graph(model: tf.keras.Model) -> NNCFGraph:
+    """
+    Convert Keras model graph to the NNCFGraph
+
+    :param model: Keras model
+    :return: NNCFGraph
+    """
+    func_model = is_functional_model(model)
+    seq_model = is_sequential_model(model)
+
+    if not func_model and not seq_model:
+        RuntimeError('convert_keras_model_to_nxmodel function supports '
+                     'only sequential or functional models')
+
+    if func_model:
+        nncf_graph = _get_nncf_graph_from_functional(model)
+    else:
+        nncf_graph = _get_nncf_graph_from_sequential(model)
+
+    return nncf_graph
+
+
+def _get_nncf_graph_from_functional(model: tf.keras.Model) -> NNCFGraph:
+    model_config = model.get_config()
+    raw_nodes = _prepare_raw_nodes(model)
+    return _get_nncf_graph_from_raw_nodes(model_config, raw_nodes)
+
+
+def _prepare_raw_nodes(model: tf.keras.layers.Layer) -> Dict:
```

Comment:
Isn't model tf.keras.Model? Layer definitely doesn't have config['layers'].

Reply:
It's a typo. Fixed.